src/app/index.ts

```diff
@@ -75,6 +87,19 @@ class WebStarter extends Generator {
       private: true,
     });
   }
+
+  /**
+   * Shortcut the promptOrUninteractive call with prefilled arguments.
+   *
+   * @param prompts
+   */
+  private async _promptOrUninteractive(prompts: Generator.Questions) {
+    return await promptOrUninteractive(
```

Comment:
You're in an async function so you don't need to await here, too.

Reply:
Removed.